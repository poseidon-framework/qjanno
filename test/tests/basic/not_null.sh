qjanno "SELECT * FROM data/basic.csv WHERE bar IS NOT NULL"
