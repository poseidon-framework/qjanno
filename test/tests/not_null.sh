qhs -H "SELECT * FROM basic.csv WHERE bar IS NOT NULL"
