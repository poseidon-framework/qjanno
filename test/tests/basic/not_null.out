.----------------.-----.-----.-----.
|  source_file   | foo | bar | baz |
:================:=====:=====:=====:
| data/basic.csv | a0  | 1   | a2  |
| data/basic.csv | b0  | 3   | b2  |
'----------------'-----'-----'-----'
